netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -47,10 +47,18 @@
         // @checkstyle LineLengthCheck (1 line)
         "(?<!\\]\\()(?<!=\")(https?:\\/\\/[a-zA-Z0-9-._~:\\?#@!$&'*+,;=%\\/]+[a-zA-Z0-9-_~#@$&'*+=%\\/])(?![\\w.]*\\]\\()"
     );
+    /**
+     * Pattern to detect lines which should have a line break on the end.
+     */
+    private static final Pattern NEW_LINE = Pattern.compile(
+        "^ {0,3}\\S* ?$", Pattern.MULTILINE
```

Comment:
@mkordas we look for lines which have less then two spaces on their end because TxtMark automatically puts `<br/>` if the line has two or more spaces on its end
and we should skip a line that starts with four or more spaces because it's a code block in markdown